File: TaskManager/app/models/entities/Progect.go
package entities

// Progect структура сущности проекта
type Progect struct {
	ID            int64  `json:"ID"`             // идентификатор
	Name    	  string `json:"name"`     		 // название
	Description   string `json:"description"`    // описание
	Fk_employee   int64  `json:"fk_employee"`    //создатель
	Employee      string `json:"employee"`    // описание
}

File: task_manager/public/js/src/models/entities/progect.js
// Employee класс для представления сущности сотрудника
export class Progect {
    constructor(id, name,description, employee,fk_employee) {
        this.ID = id
        this.name = name
        this.description=description
        this.employee = employee
        this.fk_employee=fk_employee
    }
}